### PR TITLE
feat(plugin-meetings): can invite/admit in meeting actions

### PR DESCRIPTION
**Pull request by Coread**
# COMPLETES #< INSERT LINK TO ISSUE >

## This pull request addresses

The ability to invite a guest to a meeting is controlled by the ADD_GUEST display hint.
The ability to admit a participant to a meeting is controlled by the ROSTER_WAITING_TO_JOIN display hint.

## by making the following changes

* added the new inMeetingAction canInviteGuest. I've used invite rather than add, because adding doesn't dial a user and the UI says invite guest.
* added the new inMeetingAction canAdmitParticipant.

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js:33]: should not matter much but should we keep the same name guest/ participant !?

File: packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js:15]: if you want to convert the class to ts its the right time to do :D

File: packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js
Comment (Coread) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js:33]: I wasn't sure they are always the same thing... Are you definitely admitting a guest? Or can invitees be admitted as well

File: packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js
Comment (Coread) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js]: done

File: packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js:31]: End of the day they all come as participant , So was thinking canInviteNewParticipants and stick with participant and not guest

**Review by arun3528 — APPROVED** [review]

File: packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js
Comment (Coread) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js:31]: ok sure. I've changed to canInviteNewParticipants

Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.js:31]: looks good, let me know if i need to merge , Its waiting for someone from web client to provide approval

**Review by martroos — APPROVED** [review]

**Pull request merged**
